### DISCUSSION TITLE
Regenerate Parts Function

### DISCUSSION THREAD

**Discussion by Ahh6483** [musescore/MuseScore#15661] [Ideas]
Hello there!
When I was messing around with a few of my MS3 scores and some new MS4 score I found that a lot of the time edits made in the Main Score (stem up or down, adding a hairpin or any lines, and other smaller things) would either not transfer to the Parts or would be places oddly. My fix to this would be Duplicating the part already made. I would find it very helpful if there was a Regenerate Parts Button that would give you a fresh set of parts. It would also be very helpful to be able to customize what parts it automatically makes. 

In addition is there any reason why you are not able to delete the default parts given by MS4? While I find this for the most part a lot easier, in some cases, example when writing a piano reduction with cues, I wouldn't need a whole new part for just the cues but it automatically creates one I cannot remove. 

Let me know if you have any questions,
Thank you for your consideration.

**Comment by cbjeukendrup** [comment]
Related to #15546
